config/application.yml

```diff
@@ -0,0 +1,2 @@
+gmail_username: 'hypertube.thewildcards@gmail.com'
```

Comment:
merge development in to your branch and git rm this file and re add the username and password to fix this